src/lib/ngsiNotify/Notifier.h

```diff
@@ -44,12 +44,13 @@ class Notifier
 public:
   virtual ~Notifier(void);
 
-  virtual void sendNotifyContextRequest(NotifyContextRequest* ncr,
-                                        const std::string&    url,
-                                        const std::string&    tenant,
-                                        const std::string&    xauthToken,
-                                        const std::string&    fiwareCorrelator,
-                                        NotificationFormat    notifyFormat);
+  virtual void sendNotifyContextRequest(NotifyContextRequest*            ncr,
+                                        const std::string&               url,
+                                        const std::string&               tenant,
+                                        const std::string&               xauthToken,
+                                        const std::string&               fiwareCorrelator,
+                                        RenderFormat                     renderFormat,
+                                        const std::vector<std::string>&  attrsFilter);
```

Comment:
Just a filter or it also acts as a orderding criteria? In that case attrsOrder would be better...